src/ui/component/viewers/appViewer/view.jsx

```diff
@@ -0,0 +1,61 @@
+// @flow
+import React, { useState, useEffect } from 'react';
+import LoadingScreen from 'component/common/loading-screen';
+
+type Props = {
+  source: string,
+  claim: StreamClaim,
+  contentType: string,
+};
+
+const SANDBOX_TYPES = ['application/x-lbry', 'application/x-ext-lbry'];
+
+// This server exists in src/platforms/electron/startSandBox.js
+const SANDBOX_SET_BASE_URL = 'http://localhost:5278/set/';
```

Comment:
are these important enough configuration values that they should be hoisted to top-level config of some kind? does anything else use them?

Reply:
Nope, they exist entirely for this app viewer